scripts/devops_tasks/test_run_samples.py

```diff
@@ -87,45 +109,112 @@
 }
 
 
+def run_check_call_with_timeout(
+    command_array,
+    working_directory,
+    acceptable_return_codes=[],
+    always_exit=False,
+    timeout=None,
+    pass_if_timeout=False
```

Comment:
depending on the fact that all the items in the `TIMEOUT_SAMPLES` are being set to `True` for `pass_if_timeout`,
should we consider changing the default value to `True`?
(if `TIMEOUT_SAMPLES` are for samples designed to running forever, then default value to True looks more intuitive)